docs/kbs_attestation_protocol.md

```diff
@@ -340,6 +340,8 @@ protocol, where attesters authenticate themselves against a KBS implementation.
 This leads to the second phase, during which authenticated attesters can request
 protected resources from the KBS.
 
```

Comment:
I would add the following paragraph to introduce the two authentication scheme:

```
Attesters can access protected resources by providing the KBS with either the HTTP cookie or the
Attestation Results token returned as part of the Attestation HTTP response. The former is usually
short-lived and is typically used for one-time, synchronous resources request. For longer KBS
sessions, it is more convenient to use the latter.    
```

Reply:
> The former is usually
short-lived and is typically used for one-time, synchronous resources request. For longer KBS
sessions, it is more convenient to use the latter.

This may not be accurate at the moment, but I believe this is the intention: Being able to request resources with the token, regardless of the HTTP session status.

Reply:
Thanks for the suggestion. It looks more comprehensible!